src/main/java/LittleMissHelpful/Command/AddDeadlineCommand.java

```diff
@@ -0,0 +1,53 @@
+package LittleMissHelpful.Command;
+
+import java.time.LocalDateTime;
+import java.time.LocalDate;
+import java.time.format.DateTimeParseException;
+
+import LittleMissHelpful.Storage;
+import LittleMissHelpful.TaskList;
+import LittleMissHelpful.Ui;
+import LittleMissHelpful.Exception.InvalidCommandException;
+import LittleMissHelpful.Task.Deadline;
+
```

Comment:
The header comment for the class might be missed out, will be good if it could be added in